setting-up-appsmith/aws-ami.md

```diff
@@ -0,0 +1,185 @@
+
+- Verify that the correct key pair (created in Step 2) will be used for the server.
+
+- Confirm your selection by hitting the “Launch Instances” button.
+
+The AWS Console will now begin spinning up the new server.
+![Launch](../.gitbook/assets/aws-launch.png)
+
+The process usually takes a few minutes. Use the EC2 Dashboard to check the status of the server. Once the server has launched, you will be able to obtain its public IP address from the EC2 Dashboard, as shown below: 
+
+![EC2 Detail](../.gitbook/assets/aws-ec2-detail.png)
+
+At this point, you should be able to browse to the cloud server, by entering the cloud server IP address or DNS name directly into your browser’s address bar. You should now see your web app home page as shown below:
+![Login Page](../.gitbook/assets/aws-login-page.png)
+
+
+## Step 5: Login & start using Appsmith
+```
+At the end of this step, you will have logged in to Appsmith and build your internal tools
+```
+
+To log in to the Appsmith dashboard, follow these steps:
+- You will need to register an account to log in
+  - Go to the Sign Up page, fill in your user's email & password, then click Sign Up
+  ![Appsmith Signup Page](../.gitbook/assets/aws-appsmith_signup.png)
+
+Once you completed the registration, you will be moved to the Personal Organization page. Now you can begin to create your custom app. 
+![Appsmith Dashboard Page](../.gitbook/assets/aws-appsmith_dashboard.png)
+
+For more tutorials on how to create your custom app by Appsmith, please take a look at [https://docs.appsmith.com/](https://docs.appsmith.com/)
+
+
+## Step 6: Configure custom domain for your app
+```
+At the end of this step, you should be able to access your web app by your custom domain
+```
+
+The next step is to configure your app so that it can be accessed by your custom domain. Follow these steps:
+
+- Once your instance is ready, connect to that instance (via SSH) using your key pair (Create in step 2) and the public IP of your instance (Created in step 4) via the terminal or any SSH Client that you have
+- Move to `/home/ubuntu/appsmith/scripts`
+
+- Run `configure-ssl.sh` script
+  ```
+  ./configure-ssl.sh
+  ```
+
+- You will be asked to your input domain to proceed with the configuration(Please make sure that you have mapped your domain with EC2 Instance's public IP)
+
+    ![Custom Domain](../.gitbook/assets/aws-custom-domain.png)
+
+- There will be an option for you to configure SSL for your domain
+  ![SSL](../.gitbook/assets/aws-ssl.png)
+
+At this point, you should be able to browse to the cloud server, by entering your custom domain directly into your browser’s address bar. You should be able to see your web app home page now
+![Login Page](../.gitbook/assets/aws-login-page.png)
+
+## Step 7: Find Application Credentials
```

Comment:
As a user, I think I'm confused why I need to find these credentials at all. Can you please help set up context for this in the document? A couple of lines maybe? 

Also this section also looks AWE specific, so we should link to an AWS document

Reply:
Wanted to understand - As an when Appsmith releases new versions, how should the user go about updating to them? WE should mention this somewhere in the doc

Reply:
1. Have added context on why we need the credential section. 

2. Appsmith has auto-update turned on. Added a section for that as well.